sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/reflection/KalixMethod.scala

```diff
@@ -192,14 +192,42 @@ case class SpringRestServiceMethod(
 
 case class KalixMethod(
     serviceMethod: ServiceMethod,
-    methodOptions: Seq[kalix.MethodOptions] = Seq.empty,
```

Comment:
it turns out that `kalix.MethodOptions` don't accumulate. There is at most one instance for each method and it can have eventing, acl, jwtOptions, view, and entityOptions. As per `annotations.proto`. See how deals with it the new `mergeKalixOptions` below.